### PR TITLE
Move Max31865 package to version 1.0.1

### PR DESCRIPTION
**Pull request by erikcorry**

### CODE REVIEW

File: packages/max31865-1.0.0.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ name: max31865
 description: Driver for MAX31865 RTD-to-Digital Converter, an SPI-connected digital-to-analog converter, typically used for temperature measurement.
 license: MIT
 url: github.com/toitware/max31865-driver
-version: 1.0.0
+version: 1.0.1
```
Comment (floitsch) [inline packages/max31865-1.0.0.yaml:5]: Needs the dependencies.

**Review by floitsch — CHANGES_REQUESTED** [review]

File: packages/max31865-1.0.0.yaml (hunk 1 above)
Comment (erikcorry) [reply packages/max31865-1.0.0.yaml:5]: I added the direct dependency, but not the transitive one.  Hope this is correct.

Comment (floitsch) [reply packages/max31865-1.0.0.yaml:5]: That is correct.

**Review by floitsch — APPROVED** [review]
LGTM.

**Pull request merged**